### ISSUE TITLE
lastChange fails to anything if applied to obj.prop on line 35 of demo.js

### ISSUE THREAD

**Issue by curcio** [google/qpp#3]
Selecting the same attribute in different parts of the code yields different results.

Way to reproduce: (using demo.js)

1 - Select attribute obj.prop from line 23 and click on button labeled 'lastChange' on the Querypoint Panel.
2 - Press the button in the html a couple of times, working as expected.
3 - Reload the extension
4 - Select attribute obj.prop from line 35 and click on button labeled 'lastChange' on the panel.
5 - Press the button in the html a couple of times, not working.

**Comment by johnjbarton** [comment]
Just to verify: set 3 means the reload button on the Web page is clicked correct?

**Comment by curcio** [comment]
I wasn't sure that was enough.Just in case I closed the inspector, reloaded the Web Page and opened the inspector again.

**Comment by johnjbarton** [comment]
I believe that the procedure you used is not like reload but rather starting all over.

**Comment by johnjbarton** [comment]
We get an exception on loc.traceId = traceId;

**Comment by johnjbarton** [comment]
By using break on exception we stop with a stack trace in a code transformation. By looking at the value of

```
traceur.outputgeneration.TreeWriter.write(tree)
```

for different frames in the stack trace we can see that the problem is that 

```
var operand = this.insertVariableFor(tree.operand);
```

is called on 

```
__qp_527_3.prop
```

The stack trace:

InsertVariableForExpressionTransformer.insertVariableFor (InsertVariableForExpressionTransformer.js:70)
PropertyReferenceTransformer.transformMemberExpression (ValueChangeQueryTransformer.js:128)
$MemberExpression.transform (traceur.js:4819)
$ParseTreeTransformer.transformAny (traceur.js:6181)
PropertyReferenceTransformer.transformAny (ValueChangeQueryTransformer.js:115)
ValueChangeQueryTransformer.transformBinaryOperator (ValueChangeQueryTransformer.js:237)
$BinaryOperator.transform (traceur.js:3949)
$ParseTreeTransformer.transformAny (traceur.js:6181)
ValueChangeQueryTransformer.transformAny (ValueChangeQueryTransformer.js:184)
$ParseTreeTransformer.transformExpressionStatement (traceur.js:6426)
$ExpressionStatement.transform (traceur.js:4455)
$ParseTreeTransformer.transformAny (traceur.js:6181)
ValueChangeQueryTransformer.transformAny (ValueChangeQueryTransformer.js:184)
InsertingTransformer.transformListInsertEach (InsertingTransformer.js:153)
InsertingTransformer.transformBlock (InsertingTransformer.js:87)
$Block.transform (traceur.js:4007)
$ParseTreeTransformer.transformAny (traceur.js:6181)
ValueChangeQueryTransformer.transformAny (ValueChangeQueryTransformer.js:184)
$ParseTreeTransformer.transformIfStatement (traceur.js:6509)
$IfStatement.transform (traceur.js:4682)
$ParseTreeTransformer.transformAny (traceur.js:6181)
ValueChangeQueryTransformer.transformAny (ValueChangeQueryTransformer.js:184)
InsertingTransformer.transformListInsertEach (InsertingTransformer.js:153)
InsertingTransformer.transformBlock (InsertingTransformer.js:87)
$Block.transform (traceur.js:4007)
$ParseTreeTransformer.transformAny (traceur.js:6181)
ValueChangeQueryTransformer.transformAny (ValueChangeQueryTransformer.js:184)
$ParseTreeTransformer.transformFunctionBody (traceur.js:6488)
$ParseTreeTransformer.transformFunction (traceur.js:6475)
$ParseTreeTransformer.transformFunctionDeclaration (traceur.js:6482)
$FunctionDeclaration.transform (traceur.js:4579)
$ParseTreeTransformer.transformAny (traceur.js:6181)
ValueChangeQueryTransformer.transformAny (ValueChangeQueryTransformer.js:184)
InsertingTransformer.transformListInsertEach (InsertingTransformer.js:153)
InsertingTransformer.transformProgram (InsertingTransformer.js:121)
$Program.transform (traceur.js:5130)
$ParseTreeTransformer.transformAny (traceur.js:6181)
ValueChangeQueryTransformer.transformAny (ValueChangeQueryTransformer.js:184)
Querypoint.ValueChangeQuery.transformParseTree (ValueChangeQuery.js:72)
(anonymous function) (QPProject.js:50)
(anonymous function) (QPProject.js:49)
QPProject.generateSourceFromTrees (QPProject.js:46)
$WebPageProject.runInWebPage (traceur.js:18369)
QPProject.runInWebPage (QPProject.js:77)
(anonymous function) (QPProject.js:109)
callbackWrapper (ExtensionAPI.js:707)
ExtensionServerClient._onCallback (ExtensionAPI.js:905)
ExtensionServerClient._onMessage (ExtensionAPI.js:914)

**Comment by johnjbarton** [comment]
Fix: cecbdf0c027fc6c9445604d592a9b43299769979

[ ] need test case
[ ] need to file new bug: the condition.prop value '7' is shown as the lastChange

**Comment by johnjbarton** [comment]
The test case is lastChangeOnRHS.html.  New issues #5 and #6 were filed and closed.
